content/ADR-154-multiplayer-ootb.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: adr
+adr: 154 # replace this number for the PR or ISSUE number
+title: Multiplayer out of the box
+date: 2022-12-15
+status: Idea # pick one of these
+type: RFC # pick one of these
+spdx-license: CC0-1.0
+authors:
+  - nearnsahw # this is your github username
+# remove the following line! it exists to render the template nicely
+---
+
+## Abstract
+
+<!--
+Abstract is a multi-sentence (short paragraph) technical summary. This should be a very terse and human-readable version of the document section. **Someone should be able to read only the abstract to get the gist of what this document is about in its current state.** Abstracts should be always up to date with the current state of the document.
+-->
+
+The purpose of this feature is to ,ake it easier for content creators to build scenes where what one player changes is seen and shared by all other players there. This today is possible but very hard, and because of that very few creators explore those options.
```

Comment:
```suggestion
The purpose of this feature is to make it easier for content creators to build scenes where what one player changes is seen and shared by all other players there. This today is possible but very hard, and because of that very few creators explore those options.
```